requirements.txt

```diff
@@ -1,4 +1,5 @@
 boto>=2.7
+boto3
```

Comment:
Makes sense.  Not sure where it was introduced, but I'll use 1.4.4 as that's been out for ~2 months